### ISSUE TITLE
Query Radius 

### ISSUE THREAD

**Issue by endeavorXx** [jakevdp/pyTree#2]
were you able to implement Query radius ?

**Comment by jakevdp** [comment]
Yes - this code ended up in scikit-learn, and there is a query_radius method: https://scikit-learn.org/dev/modules/generated/sklearn.neighbors.KDTree.html#sklearn.neighbors.KDTree.query_radius

**Comment by endeavorXx** [comment]
so, does KDTree implement your code or just the query radius method?

**Comment by jakevdp** [comment]
The code here eventually became the KDTtee in scikit-learn

**Comment by endeavorXx** [comment]
It's cool 
You are great !!
Why this KD-Tree algorithm does not work well on high dimensional data compared to Ball Tree.

**Comment by endeavorXx** [comment]
Did you code it in a way that makes it more robust only up to the dimension of 3?
